source/release-notes/release_3_9_0.rst

```diff
@@ -0,0 +1,181 @@
+
+        * Now **Syscollector** gets *bonded* interfaces' MAC:
+
+            This change adds the possibility of getting the real MAC address of each interface in `/sys/class/net/address` instead of getting it from interfaces with *AF_PACKET* sockets, avoiding this way problems with bonded interfaces that share the same MAC address at the software level.
+
+        * From this version on, the *manager* will know the primary IP of the agents that will be updated every time an agent sends a keep alive.
+
+            The manager will store that IP in the global database and it will be printed in the Wazuh app. With this change, although an agent is registered with **any** as IP, the manager will know the agent's current IP. 
+
+        * Added support to AWS organizations in *CloudTrail* service:
+
+            With this enhancement, it is possible getting logs for created organizations by adding ``<aws_organization_id>ORGANIZATION</aws_organization_id>`` in the wodle configuration, which will create a new directory in the path where save the *accounts id*.
+
+            .. code-block:: xml
+
+                <wodle name="aws-s3">
+                    <disabled>no</disabled>
+                    <bucket type="cloudtrail">
+                        <name>cloudtrail</name>
+                        <aws_organization_id>wazuh</aws_organization_id>
+                        <aws_profile>default</aws_profile>
+                    </bucket>
+                    <remove_from_bucket>no</remove_from_bucket>
+                    <interval>20m</interval>
+                    <run_on_start>yes</run_on_start>
+                    <skip_on_error>no</skip_on_error>
+                </wodle>
+
+        * Optimized network performance in *ossec-remoted* daemon by introducing a network buffer to cache incomplete messages from agents.
+
+            With this method, *ossec-remoted* daemon does not wait for complete messages when it's not necessary, improving the performance of the network greatly.
+
+        * Labels starting with ``_`` are reserved now for internal use only (this was added to allow the manager to know the agent's IP upper mentioned).
+
+        * Improved cluster performance: Embedded *Python 3* and updated to *asyncio library*:
+
+            Changed the internal cluster structure to adapt it to the 3.7.2 version of Python, which along with the *asyncio library* (`Asynchronous I/O <https://docs.python.org/3/library/asyncio.html>`_) has improved the cluster speed greatly. 
+
+Fixes
+^^^^^
+
+        - Fixed error in Syscollector for Windows older than Vista when gathering the hardware inventory.
+        - Fixed an error in the OSquery configuration validation.
+        - Wazuh manager starts regardless of the contents of ``local_decoder.xml``.
+        - Delete empty fields from Windows Eventchannel alerts.
+        - Fixed memory leak and crash in Vulnerability Detector.
+        - ...
```

Comment:
I think we should not use "...", and also extend this list with the list of fixes (https://github.com/wazuh/wazuh/blob/3.9/CHANGELOG.md#fixed)

Reply:
Done @vikman90 , thanks for the feedback :)